hugegraph-api/src/main/java/com/baidu/hugegraph/config/ServerOptions.java

```diff
@@ -250,4 +250,12 @@ public static synchronized ServerOptions instance() {
                     null,
                     "hugegraph"
             );
+
+    public static final ConfigOption<Boolean>  DYNAMIC_LISTEN_GRAPH_CHANGES =
+            new ConfigOption<>(
+                    "dynamic_create_graph",
```

Comment:
can we rename to "graphs.disable_dynamic_create_drop" and update the description?